src/styxdocker/__init__.py

```diff
@@ -171,6 +174,7 @@ def __init__(
         self.data_dir = pathlib.Path(data_dir or "styx_tmp")
         self.uid = os.urandom(8).hex()
         self.execution_counter = 0
+        self.userid = os.getuid() if os.name == "posix" else None
```

Comment:
Maybe expose a constructor argument in the Runner to let users override this.

Reply:
Should this sitll try to get the userif id none provided (`os.getuid() if os.name == "posix" else None`) if none is provided or should we always let the user set it themselves?

Reply:
Yes still getuid on linux

Reply:
In that case, this should be good for another look.